README.md

```diff
@@ -140,6 +140,10 @@ If you want to add annotators/annotation approvers, see [Frequently Asked Questi
 > [^1]: (1) EC2 KeyPair cannot be created automatically, so make sure you have an existing EC2 KeyPair in one region. Or [create one yourself](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-key-pairs.html#having-ec2-create-your-key-pair). (2) If you want to access doccano via HTTPS in AWS, here is an [instruction](https://github.com/doccano/doccano/wiki/HTTPS-setting-for-doccano-in-AWS).
 > [^2]: Although this is a very cheap option, it is only suitable for very small teams (up to 80 concurrent requests). Read more on [Cloud Run docs](https://cloud.google.com/run/docs/concepts).
 
+## Documentation
+
+See [here](./docs).
```

Comment:
In fact, doccano has GitHub pages which is automatically generated from docs directory. So the following link is better:
- https://doccano.github.io/doccano/

Reply:
Done.